spec/spec_helper.rb

```diff
@@ -72,7 +73,7 @@
   # Print the 10 slowest examples and example groups at the
   # end of the spec run, to help surface which specs are running
   # particularly slow.
-  config.profile_examples = 10
+  # config.profile_examples = 10
```

Comment:
You can achieve this by:

``` sh
echo '--fail-fast --no-profile' >> .rspec-local

Reply:
Oops. I didn't mean to commit this.

Reply:
I got this. :D Just saying that you can use `.rspec-local` to tune up rspec to your tastes :D